Installtion of docker-compose fails due to missing compiler

The following error occurred when installing docker-compose:
```
TASK [phrosenberg.docker : Install Docker Compose] ********************************************************************************************************************************************************************************************
fatal: [kafka_dev_01]: FAILED! => changed=false 
  cmd:
  - /bin/pip2
  - install
  - docker-compose
  msg: |-
    stdout: Collecting docker-compose
      Using cached https://files.pythonhosted.org/packages/2e/93/b8fb6532487fcc40f5c607ac428a609e7f74bfb26a1c3c980a253c6e5a14/docker_compose-1.25.0-py2.py3-none-any.whl
    Requirement already satisfied: backports.shutil-get-terminal-size==1.0.0; python_version < "3.3" in /usr/lib/python2.7/site-packages (from docker-compose)
    Collecting subprocess32<4,>=3.5.4; python_version < "3.2" (from docker-compose)
      Using cached https://files.pythonhosted.org/packages/32/c8/564be4d12629b912ea431f1a50eb8b3b9d00f1a0b1ceff17f266be190007/subprocess32-3.5.4.tar.gz
    Requirement already satisfied: docker[ssh]<5,>=3.7.0 in /usr/lib/python2.7/site-packages (from docker-compose)
    Requirement already satisfied: PyYAML<5,>=3.10 in /usr/lib64/python2.7/site-packages (from docker-compose)
    Requirement already satisfied: jsonschema<4,>=2.5.1 in /usr/lib/python2.7/site-packages (from docker-compose)
    Collecting texttable<2,>=0.9.0 (from docker-compose)
      Using cached https://files.pythonhosted.org/packages/82/a8/60df592e3a100a1f83928795aca210414d72cebdc6e4e0c95a6d8ac632fe/texttable-1.6.2.tar.gz
    Requirement already satisfied: websocket-client<1,>=0.32.0 in /usr/lib/python2.7/site-packages (from docker-compose)
    Requirement already satisfied: requests<3,>=2.20.0 in /usr/lib/python2.7/site-packages (from docker-compose)
    Collecting cached-property<2,>=1.2.0 (from docker-compose)
      Using cached https://files.pythonhosted.org/packages/3b/86/85c1be2e8db9e13ef9a350aecd6dea292bd612fa288c2f40d035bb750ded/cached_property-1.5.1-py2.py3-none-any.whl
    Requirement already satisfied: enum34<2,>=1.0.4; python_version < "3.4" in /usr/lib/python2.7/site-packages (from docker-compose)
    Requirement already satisfied: backports.ssl-match-hostname<4,>=3.5; python_version < "3.5" in /usr/lib/python2.7/site-packages (from docker-compose)
    Collecting docopt<1,>=0.6.1 (from docker-compose)
      Using cached https://files.pythonhosted.org/packages/a2/55/8f8cab2afd404cf578136ef2cc5dfb50baa1761b68c9da1fb1e4eed343c9/docopt-0.6.2.tar.gz
    Requirement already satisfied: six<2,>=1.3.0 in /usr/lib/python2.7/site-packages (from docker-compose)
    Requirement already satisfied: ipaddress<2,>=1.0.16; python_version < "3.3" in /usr/lib/python2.7/site-packages (from docker-compose)
    Collecting dockerpty<1,>=0.4.1 (from docker-compose)
      Using cached https://files.pythonhosted.org/packages/8d/ee/e9ecce4c32204a6738e0a5d5883d3413794d7498fe8b06f44becc028d3ba/dockerpty-0.4.1.tar.gz
    Collecting paramiko>=2.4.2; extra == "ssh" (from docker[ssh]<5,>=3.7.0->docker-compose)
      Using cached https://files.pythonhosted.org/packages/4b/80/74dace9e48b0ef923633dfb5e48798f58a168e4734bca8ecfaf839ba051a/paramiko-2.6.0-py2.py3-none-any.whl
    Requirement already satisfied: repoze.lru in /usr/lib/python2.7/site-packages (from jsonschema<4,>=2.5.1->docker-compose)
    Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/lib/python2.7/site-packages (from requests<3,>=2.20.0->docker-compose)
    Requirement already satisfied: certifi>=2017.4.17 in /usr/lib/python2.7/site-packages (from requests<3,>=2.20.0->docker-compose)
    Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /usr/lib/python2.7/site-packages (from requests<3,>=2.20.0->docker-compose)
    Requirement already satisfied: idna<2.9,>=2.5 in /usr/lib/python2.7/site-packages (from requests<3,>=2.20.0->docker-compose)
    Collecting bcrypt>=3.1.3 (from paramiko>=2.4.2; extra == "ssh"->docker[ssh]<5,>=3.7.0->docker-compose)
      Using cached https://files.pythonhosted.org/packages/ad/36/9a0227d048e98409f012570f7bef8a8c2373b9c9c5dfbf82963cbae05ede/bcrypt-3.1.7-cp27-cp27mu-manylinux1_x86_64.whl
    Collecting pynacl>=1.0.1 (from paramiko>=2.4.2; extra == "ssh"->docker[ssh]<5,>=3.7.0->docker-compose)
      Using cached https://files.pythonhosted.org/packages/b3/25/e605574f24948a8a53b497744e93f061eb1dbe7c44b6465fc1c172d591aa/PyNaCl-1.3.0-cp27-cp27mu-manylinux1_x86_64.whl
    Collecting cryptography>=2.5 (from paramiko>=2.4.2; extra == "ssh"->docker[ssh]<5,>=3.7.0->docker-compose)
      Using cached https://files.pythonhosted.org/packages/e2/67/4597fc5d5de01bb44887844647ab8e73239079dd478c35c52d58a9eb3d45/cryptography-2.8-cp27-cp27mu-manylinux1_x86_64.whl
    Requirement already satisfied: cffi>=1.1 in /usr/lib64/python2.7/site-packages (from bcrypt>=3.1.3->paramiko>=2.4.2; extra == "ssh"->docker[ssh]<5,>=3.7.0->docker-compose)
    Requirement already satisfied: pycparser in /usr/lib/python2.7/site-packages (from cffi>=1.1->bcrypt>=3.1.3->paramiko>=2.4.2; extra == "ssh"->docker[ssh]<5,>=3.7.0->docker-compose)
    Installing collected packages: subprocess32, texttable, cached-property, docopt, dockerpty, docker-compose, bcrypt, pynacl, cryptography, paramiko
      Running setup.py install for subprocess32: started
        Running setup.py install for subprocess32: finished with status 'error'
        Complete output from command /usr/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-n96cGd/subprocess32/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-NyjgHF-record/install-record.txt --single-version-externally-managed --compile:
        running install
        running build
        running build_py
        creating build
        creating build/lib.linux-x86_64-2.7
        copying subprocess32.py -> build/lib.linux-x86_64-2.7
        running build_ext
        running build_configure
        checking for gcc... no
        checking for cc... no
        checking for cl.exe... no
        configure: error: in `/tmp/pip-build-n96cGd/subprocess32':
        configure: error: no acceptable C compiler found in $PATH
        See `config.log' for more details
        Traceback (most recent call last):
          File "<string>", line 1, in <module>
          File "/tmp/pip-build-n96cGd/subprocess32/setup.py", line 120, in <module>
            main()
          File "/tmp/pip-build-n96cGd/subprocess32/setup.py", line 114, in main
            'Programming Language :: Python :: Implementation :: CPython',
          File "/usr/lib/python2.7/site-packages/setuptools/__init__.py", line 129, in setup
            return distutils.core.setup(**attrs)
          File "/usr/lib64/python2.7/distutils/core.py", line 151, in setup
            dist.run_commands()
          File "/usr/lib64/python2.7/distutils/dist.py", line 953, in run_commands
            self.run_command(cmd)
          File "/usr/lib64/python2.7/distutils/dist.py", line 972, in run_command
            cmd_obj.run()
          File "/usr/lib/python2.7/site-packages/setuptools/command/install.py", line 61, in run
            return orig.install.run(self)
          File "/usr/lib64/python2.7/distutils/command/install.py", line 563, in run
            self.run_command('build')
          File "/usr/lib64/python2.7/distutils/cmd.py", line 326, in run_command
            self.distribution.run_command(command)
          File "/usr/lib64/python2.7/distutils/dist.py", line 972, in run_command
            cmd_obj.run()
          File "/usr/lib64/python2.7/distutils/command/build.py", line 127, in run
            self.run_command(cmd_name)
          File "/usr/lib64/python2.7/distutils/cmd.py", line 326, in run_command
            self.distribution.run_command(command)
          File "/usr/lib64/python2.7/distutils/dist.py", line 972, in run_command
            cmd_obj.run()
          File "/tmp/pip-build-n96cGd/subprocess32/setup.py", line 41, in run
            self.run_command(command)
          File "/usr/lib64/python2.7/distutils/cmd.py", line 326, in run_command
            self.distribution.run_command(command)
          File "/usr/lib64/python2.7/distutils/dist.py", line 972, in run_command
            cmd_obj.run()
          File "/tmp/pip-build-n96cGd/subprocess32/setup.py", line 26, in run
            raise RuntimeError(configure_command + ' failed.')
        RuntimeError: sh ./configure failed.
  
        ----------------------------------------
  
    :stderr: WARNING: Running pip install with root privileges is generally not a good idea. Try `pip2 install --user` instead.
    Command "/usr/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-n96cGd/subprocess32/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-NyjgHF-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-n96cGd/subprocess32/
```

Installing package `gcc` solved the problem.

The role should ensure that `gcc` is installed.

--- a/tasks/_packages.yml
+++ b/tasks/_packages.yml
@@ -2,7 +2,7 @@
 - name: Install Additional Prerequisites
   package:
     name: "{{ item }}"
-  loop: "{{ docker_additional_packages }}"
+  loop: "{{ (docker_additional_packages + docker_prerequisite_packages) | unique }}"
 
 - name: Deploy Docker CE Repository  # noqa 305
   shell: "dnf config-manager --add-repo={{ docker_ce_repo_url }}"

--- a/vars/main.yml
+++ b/vars/main.yml
@@ -1,4 +1,6 @@
 ---
+docker_prerequisite_packages:
+  - gcc  # installation of docker-compose requires gcc
 supported_ansible_distribution:
   - Amazon
   - CentOS

